models/chaincodes.js

```diff
@@ -0,0 +1,20 @@
+const firestore = require('../utils/firestore')
+const admin = require('firebase-admin')
+
+const chaincodeModel = firestore.collection('chaincodes')
+const storeChaincode = async (username) => {
+  try {
+    const userchaincode = await chaincodeModel.add({

```

Comment:
Please use camelCase for variable naming
https://javascript.info/variables#variable-naming
```suggestion
    const userChaincode = await chaincodeModel.add({

```